{"uuid":"608a4fc7-08f4-45be-85a6-6d98435e77ad","historyId":"693f3b1cc3db2de0dd58627345cb6bd1","fullName":"Array: The user is able to submit code in tryEditor for Search the array link","labels":[{"name":"tag","value":"arrays"},{"name":"host","value":"LAPTOP-RSES787T"},{"name":"thread","value":"57156@LAPTOP-RSES787T.main(1)"},{"name":"feature","value":"Array"},{"name":"story","value":"The user is able to submit code in tryEditor for Search the array link"},{"name":"suite","value":"Array"},{"name":"testClass","value":"The user is able to submit code in tryEditor for Search the array link"},{"name":"framework","value":"cucumber4jvm"},{"name":"language","value":"java"},{"name":"gherkin_uri","value":"file:///C:/Users/valli/Git/DS-ALGO/src/test/resources/features/5arrays.feature"},{"name":"package","value":"src.test.resources.features.5arrays_feature.Array"}],"links":[],"name":"The user is able to submit code in tryEditor for Search the array link","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"Result do not match expected [submission success] but found [Error occurred during submission]","trace":"java.lang.AssertionError: Result do not match expected [submission success] but found [Error occurred during submission]\r\n\tat org.testng.Assert.fail(Assert.java:110)\r\n\tat org.testng.Assert.failNotEquals(Assert.java:1413)\r\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:149)\r\n\tat org.testng.Assert.assertEquals(Assert.java:131)\r\n\tat org.testng.Assert.assertEquals(Assert.java:655)\r\n\tat com.stepDefinations.ArraysStepDef.the_user_should_be_presented_with_successful_submission_message(ArraysStepDef.java:160)\r\n\tat ✽.The user should be presented with successful submission message(file:///C:/Users/valli/Git/DS-ALGO/src/test/resources/features/5arrays.feature:110)\r\n"},"stage":"finished","steps":[{"name":"Given  The user is on \"Question\" page of \"Search the array\" after logged in","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673378283922,"stop":1673378284125},{"name":"When  The user enter valid python code in tryEditor from sheet \"pythonCode\" and 3 for the question","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673378284126,"stop":1673378285172},{"name":"And  The user clicks on Submit button","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673378285172,"stop":1673378285232},{"name":"Then  The user should be presented with successful submission message","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"Result do not match expected [submission success] but found [Error occurred during submission]","trace":"java.lang.AssertionError: Result do not match expected [submission success] but found [Error occurred during submission]\r\n\tat org.testng.Assert.fail(Assert.java:110)\r\n\tat org.testng.Assert.failNotEquals(Assert.java:1413)\r\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:149)\r\n\tat org.testng.Assert.assertEquals(Assert.java:131)\r\n\tat org.testng.Assert.assertEquals(Assert.java:655)\r\n\tat com.stepDefinations.ArraysStepDef.the_user_should_be_presented_with_successful_submission_message(ArraysStepDef.java:160)\r\n\tat ✽.The user should be presented with successful submission message(file:///C:/Users/valli/Git/DS-ALGO/src/test/resources/features/5arrays.feature:110)\r\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673378285232,"stop":1673378286910}],"attachments":[],"parameters":[{"name":"Sheetname","value":"pythonCode"},{"name":"RowNumber","value":"3"}],"start":1673378283921,"stop":1673378287235}